### ISSUE TITLE
Test for different masking hyperparameter values

### ISSUE THREAD

**Issue by vmazashvili** [vmazashvili/Neural-Networks#4]
Test for different masking hyperparameter values: Define a list of masking thresholds
Generate masked dataset and related information using mask_dataset function
Fine-tuning the model with current masked dataset
Save the fine-tuned model with a filename indicating the threshold used

**Comment by vmazashvili** [comment]
Implemented, but needs optimization
